packages/vector_graphics_compiler/lib/vector_graphics_compiler.dart

```diff
@@ -253,3 +258,18 @@ Future<Uint8List> encodeSvg(String input, String filename) async {
   }
   return buffer.done().buffer.asUint8List();
 }
+
+/// Arguement configuration object for encodeSvg function.
+class ArgConfig {
```

Comment:
This would need a more descriptive name to keep it, but maybe this just needs to be named parameters on `encodeSvg` instead of an object-  that will make more sense if we're not passing this around.

Reply:
fixed.